hack/jenkins/common.sh

```diff
@@ -31,7 +31,7 @@ export KUBECONFIG="${TEST_HOME}/kubeconfig"
 export PATH=$PATH:"/usr/local/bin/:/usr/local/go/bin/:$GOPATH/bin"
 
 # installing golang so we could do go get for gopogh
-sudo ./installers/check_install_golang.sh "1.13.4" "/usr/local" || true
+sudo ./installers/check_install_golang.sh "1.13.7" "/usr/local" || true
```

Comment:
there is another one also, in 

hack/jenkins/minikube_cross_build_and_upload.sh

Reply:
Thanks :100:

Reply:
this file already retrieved goland version from Makefile in command `WANT_GOLANG_VERSION=$(grep '^GO_VERSION' Makefile | awk '{ print $3 }')`

Reply:
I move golang upgrade to another PR